README.md

```diff
@@ -96,7 +96,13 @@ An example that specifies a value for `severityThreshold` as medium and configur
 
 ### Snyk Code Testing Example
 
-Snyk Code applicable Severity Threshold: high, medium or low (default). This is specified with `codeSeverityThreshold`. This configuration fails the build when high severity issues are found, but all issues detected at medium, high would be displayed at Task console log.
+Snyk Code applicable Severity Threshold: high, medium or low (default). This is specified with `codeSeverityThreshold`. Its `failOnThreshold` if specified has to set its severity to be matching or higher severity than its `codeSeverityThreshold`. Its `failOnIssues` as defaulted to true will subsequently fail the build if issues are found to be of level corresponding or higher than `failOnThreshold` severity. If `failOnIssues` is set to false, the build will continue.
+
+| Issue level | SeverityThreshold |
+| ----------- | ----------------- |
+| error       | high              |
+| warning     | medium            |
+| note        | low               |
```

Comment:
This is confusing to me. 

-  What is Issue Level vs. SeverityThreshold? 
- Why do we need to couple the two parameters? Interdependent parameters increase the complexity and I suggest to not do this.

Reply:
then i think it would be good to leave this table mapping out of README.md. This issue level refers to the `level` property in the snyk code output json. This json property mapping could be abstracted away without mention at README.